src/plugins/intel_cpu/src/.clang-tidy

```diff
@@ -44,6 +44,9 @@ Checks: >
   -cppcoreguidelines-narrowing-conversions,
   -cppcoreguidelines-pro-bounds-pointer-arithmetic,
   -google-build-using-namespace,
+  -google-default-arguments,
+  -google-explicit-constructor,
+  -google-readability-casting,
```

Comment:
Please add them to the comment to the list of checks we want to enable later

Reply:
Done